### PR TITLE
Add nginx tests and other related improvements

### PR DESCRIPTION
**Pull request by pombredanne**
This PR adds nginx importer and improver tests

Along the way there are several other adjustments and refinements that have been applied

https://github.com/nexB/vulnerablecode/issues/643
Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

**Comment by pombredanne** [comment]
We review these changes extensively in weekly call today. I would like to merge this ASAP unless there is an objection

File: vulnerabilities/helpers.py [hunk 1]
```diff
@@ -233,12 +216,55 @@ def get_item(object: dict, *attributes):
     >>> assert(get_item({'a': {'b': {'c': 'd'}}}, 'a', 'b', 'e')) == None
     """
     if not object:
-        LOGGER.error(f"Object is empty: {object}")
+        logger.error(f"Object is empty: {object}")
```
Comment (TG1999) [inline vulnerabilities/helpers.py:219]: Let's return from here only as discussed and remove the log
```suggestion
        return
```

**Comment by pombredanne** [comment]
Actually the nginx improver is a problem:
```
Import:31

vulnerabilities.improvers.default.DefaultImprover several times"
In [13]: f"{Vulnerability.objects.count()}, {Package.objects.count()}"
Out[13]: '31, 78'

vulnerabilities.importers.nginx.NginxBasicImprover once:
In [30]: f"{Vulnerability.objects.count()}, {Package.objects.count()}"
Out[30]: '31, 662'

```

File: vulnerabilities/helpers.py (hunk 1 above)
Comment (pombredanne) [reply vulnerabilities/helpers.py:219]: Done.

**Comment by pombredanne** [comment]
All green now .. merging !

**Review by TG1999 — APPROVED** [review]
LGTM!

**Pull request merged**